### ISSUE TITLE
UAF 1.1: P-2 FASM-API-Android-1

### ISSUE THREAD

**Issue by mlourie** [fido-alliance/conformance-tool-issues#30]
We are seeing the same issue here as for 1.0 P-2 test. This was fixed in 1.0.

Our get info response:

```
{"responseData":{"Authenticators":[{"aaid":"4e4e#4092","asmVersions":[{"major":1,"minor":0},{"major":1,"minor":1}],"assertionScheme":"UAFV1TLV","attachmentHint":1,"attestationTypes":[15880],"authenticationAlgorithm":8,"authenticatorIndex":0,"description":"Silent Authenticator for Android","hasSettings":false,"icon":"data:image/png;base64,iVBYII=","isRoamingAuthenticator":false,"isSecondFactorOnly":false,"isUserEnrolled":true,"keyProtection":6,"matcherProtection":1,"supportedExtensionIDs":[],"tcDisplay":1,"tcDisplayContentType":"text/plain","title":"Silent ASM","userVerification":512}]},"statusCode":0}
```

Assertion: AssertionError: INVALID_TRANSACTION_CONTENT: should be <=0: expected false to be true

**Comment by jenkay** [comment]
For P-2 the request is as follows:

{"asmVersion":{"major":1,"minor":0},"requestType":"GetInfo"}

note the asmVersion field which conflicts with this:

_For a GetInfo request, the following ASMRequest member(s) must have the following value(s). The remaining ASMRequest members should be omitted: ASMRequest.requestType must be set to GetInfo_

for the response we return (abbreviated)

{"responseData":{"Authenticators":[{"aaid":"4e4e#4090","asmVersions":[**{"major":1,"minor":0},{"major":1,"minor":1}**],"assertionScheme": …

but this results in the following failure:

AssertionError: INVALID_TRANSACTION_CONTENT: Should be <= 0: expected false to be true

If we remove the 1.1 version from the response, i.e. {"major":1,"minor":1}, the minor field is <=0 and the test passes.

{"Authenticators":[{"aaid":"4e4e#4090","asmVersions":[**{"major":1,"minor":0}**],"assertionScheme”: …

Note this is a UAF 1.1 test, so the ASM should return 1.1

**Comment by herrjemand** [comment]
Resolved in v0.8.33 (out in few hours)
